### ISSUE TITLE
Add an "originally built" date to rides

### ISSUE THREAD

**Issue by rctdude2** [OpenRCT2/OpenRCT2#7761]
**OS:** n/a
**Version:** 0.2.0
**Commit/Build:** tested on 9e559d5, but all builds since the refurbish feature was added

The new refurbish feature resets the reliability and build date of rides. Because it resets the build date, the original date is lost upon refurbishment. An "originally built" date would be nice to have.
I imagine this would require the new save format.

- [ ] Reproducible in RCT2 (vanilla)?
- [ ] Specific to multiplayer?

**Steps to reproduce:**
1. Start any scenario with old rides (e.g. Sprightly Park)
2. Refurbish a ride

**Dump file**
n/a

**Screenshots / Video:**
https://www.youtube.com/watch?v=2M_YaRr6wCE

**Save game:**
n/a

**Comment by Gymnasiast** [comment]
It's something that I want as well. And it indeed requires a new save format.

**Comment by Patrik356b** [comment]
@Gymnasiast couldn't the refurb option just reset reliability and ignore the date update?

still, i'd like to see this as well

**Comment by Gymnasiast** [comment]
No. Reliability decreases faster if the ride is older. Setting it back to 100% only helps a little bit.

**Comment by Dracheron** [comment]
an idea I just spotted in here (and should remain as part of the same issue) is that you could get two types of refurb - one that just resets the reliability and one that turns back time (the one that just resets reliability should be considerably cheaper than the full refurbishment where reliability degrading resets).  This would be very useful on rides that have stood the test of time with guests and are a favorite of many guests, even if they're not not new rides.
